crates/bevy_ui/src/layout/ui_surface.rs

```diff
@@ -1,16 +1,16 @@
-use core::fmt;
-
-use taffy::TaffyTree;
-
+use crate::{layout::convert, LayoutContext, LayoutError, Node, NodeMeasure};
+#[cfg(feature = "bevy_text")]
+use crate::{Measure, MeasureArgs};
```

Comment:
Measure funcs are needed for other intrinsically sized content like images, not only text.

```suggestion
use crate::{Measure, MeasureArgs};
```